storybook/stubs/shared/stores/RootStore.qml

```diff
@@ -28,4 +28,8 @@ QtObject {
     property var getChainShortNamesForSavedWalletAddress
     property var getGasEthValue
     property var getNetworkLayer
+
+    function copyToClipboard(text) {
```

Comment:
General note, not for this pr: looks like a good candidate for moving it to cpp and make independent from the backend